### PR TITLE
WMCO-491: Updated the log collection command to easy-to-read syntax.

### PR DESCRIPTION
**Pull request by EricPonvelle**
This pull request revises the WMCO logging command to be a bit more streamlined.

GitHub: https://github.com/openshift/windows-machine-config-operator/pull/491

Version: 4.7+

Preview: https://deploy-preview-33979--osdocs.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#investigating-why-windows-machine-compute-node_troubleshooting-windows-container-workload-issues

### CODE REVIEW

**Review by aravindhp — APPROVED** [review]
/lgtm

**Comment by aravindhp** [comment]
Please backport to release-4.7 and release-4.8

File: modules/investigating-why-windows-machine-compute-node.adoc [hunk 1]
```diff
@@ -18,5 +18,5 @@ There are various reasons why a Windows Machine does not become a compute node.
 +
 [source,terminal]
 ----
-$ oc logs -f $(oc get pods -o jsonpath={.items[0].metadata.name} -n openshift-windows-machine-config-operator) -n openshift-windows-machine-config-operator
+$ oc logs -f deployment/windows-machine-config-operator) -n openshift-windows-machine-config-operator
```
Comment (mburke5678) [inline modules/investigating-why-windows-machine-compute-node.adoc:21]: @EricPonvelle I think you need to remove the closing parenthesis after _operator_, based on the linked MD file.

`$ oc logs -f deployment/windows-machine-config-operator -n openshift-windows-machine-config-operator
`

**Pull request merged**